2024-02-17 17:14:36,986 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-17 17:16:23,676 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-17 17:34:39,881 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-17 17:34:58,730 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-17 18:17:04,253 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-17 18:32:28,388 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-18 19:14:03,172 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-18 19:17:24,588 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-18 19:23:10,481 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-18 19:38:20,156 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-18 19:40:18,437 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-18 19:50:21,347 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-18 20:02:22,086 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-18 20:03:39,354 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-18 20:06:22,270 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:45]
2024-02-19 04:17:09,557 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 04:18:25,610 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 04:54:54,641 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 04:54:54,835 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 04:54:54,835 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 04:58:18,168 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 04:58:18,340 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 04:58:18,340 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:05:55,245 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:05:55,385 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:05:55,385 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:14:13,933 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:14:14,091 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:14:14,091 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:22:22,464 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:22:22,620 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:22:22,620 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:24:21,143 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:24:49,951 ERROR: Exception on /reset_password_request [GET] [in C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py:828]
Traceback (most recent call last):
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\app\routes.py", line 180, in reset_password_request
    return render_template('reset_password_request.html',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\jinja2\environment.py", line 1301, in render
    self.environment.handle_exception()
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\user\Desktop\daveblog\app\templates\reset_password_request.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\user\Desktop\daveblog\app\templates\base.html", line 31, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\app\templates\reset_password_request.html", line 8, in block 'content'
    {{ form.password.label }}<br>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\jinja2\environment.py", line 485, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'app.forms.ResetPasswordRequestForm object' has no attribute 'password'
2024-02-19 05:29:40,073 INFO: Microblog startup [in C:\Users\user\Desktop\daveblog\app\__init__.py:47]
2024-02-19 05:29:46,513 ERROR: Exception on /reset_password_request [GET] [in C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py:828]
Traceback (most recent call last):
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py", line 1463, in wsgi_app
    response = self.full_dispatch_request()
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py", line 872, in full_dispatch_request
    rv = self.handle_user_exception(e)
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py", line 870, in full_dispatch_request
    rv = self.dispatch_request()
         ^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\app.py", line 855, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\app\routes.py", line 180, in reset_password_request
    return render_template('reset_password_request.html',
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\templating.py", line 150, in render_template
    return _render(app, template, context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\flask\templating.py", line 131, in _render
    rv = template.render(context)
         ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\jinja2\environment.py", line 1301, in render
    self.environment.handle_exception()
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\jinja2\environment.py", line 936, in handle_exception
    raise rewrite_traceback_stack(source=source)
  File "C:\Users\user\Desktop\daveblog\app\templates\reset_password_request.html", line 1, in top-level template code
    {% extends "base.html" %}
  File "C:\Users\user\Desktop\daveblog\app\templates\base.html", line 31, in top-level template code
    {% block content %}{% endblock %}
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\app\templates\reset_password_request.html", line 8, in block 'content'
    {{ form.password.label }}<br>
    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\Desktop\daveblog\venv\Lib\site-packages\jinja2\environment.py", line 485, in getattr
    return getattr(obj, attribute)
           ^^^^^^^^^^^^^^^^^^^^^^^
jinja2.exceptions.UndefinedError: 'app.forms.ResetPasswordRequestForm object' has no attribute 'password'
